index.html

```diff
@@ -116,6 +116,291 @@ <h5>Skills</h5>
 
   </section>
 
+
+  <section class="work-section">
+
+    <div class="container2">
+
+      <div class="work">
+
+        <div class="arrow">
+
+          <div class="arrow1">
+            <img src="images/Vector5.png" alt="horizontal">
+          </div>
+
+          <div class="arrow2">
+            <img src="images/Vector6.png" alt="arrow-head">
+          </div>
+        </div>
+
+        <div class="work-header">
+          <h2>Work</h2>
+        </div>
+
+      </div>
+
+
+      <div class="recent-work">
+
+        <div class="recent-work-header">
+
+          <h1>My recent work</h1>
+
+        </div>
+
+        <div class="vector">
+          <img src="images/Vector4.png" alt="vector4">
+        </div>
+
+      </div>
+
+
+      <section class="multi-post-stories">
+
+        <div class="post-story">
+
+          <div class="image-placeholder"></div>
+
+          <div class="story">
+
+            <div class="post-title">
+              <h2>Multi-Post Stories</h2>
+            </div>
+
+            <div class="story-paragraph">
+
+              <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the
+                industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.
+              </p>
+
+            </div>
+
+            <div class="story-languages">
+
+              <p>CSS <img class="rectangle" src="images/Rectangle70.png" alt="rectangle">
+
+                Html <img class="rectangle" src="images/Rectangle70.png" alt="rectangle">
+
+                Bootstrap <img class="rectangle" src="images/Rectangle70.png" alt="rectangle">
+
+                Ruby
+
+              </p>
```

Comment:
- [ ] Good job here, your design looks great, however, it is important to use `ul` and `li` tags to list these technologies e.g:

```
<ul>
    <li>CSS</li>
    <li>HTML</li>
    <li>Ruby</li>
</ul>

```